LumpNav/LumpNav.py

```diff
@@ -228,7 +285,7 @@ def setupScene(self): #applet specific
     if not self.cauteryTipToCautery:
       self.cauteryTipToCautery=slicer.vtkMRMLLinearTransformNode()
       self.cauteryTipToCautery.SetName("CauteryTipToCautery")
-      m = self.readTransformFromSettings('CauteryTipToCautery')
+      m = self.readTransformFromSettings('Configurations/' + self.parameterNode.GetParameter('Configuration') + '/CauteryTipToCautery') 

```

Comment:
Okay, I will make a pull request to the GuideletLib then and make sure that this is done on that side of the appilcation instead of on the LumpNav side.

But, just to make sure I understood you correctly:
1. It was okay adding the default configuration in LumpNavWidget?
2. It was okay to have that default configuration as a selectable option in the config selector combo box (as default)? 
3. The user is meant to add new configurations by typing them into the Slicer.ini manually?
4. If there a parameter cannot be found, in the custom configuration, it should be read from the default one (see 1.)?